.github/workflows/pr.yml

```diff
@@ -63,6 +63,260 @@ jobs:
       - name: Git (Windows)
         run: npx gulp initializeSubmodulesWindowsAgent
         working-directory: ./Package
-      - name: Gulp (Windows)
-        run: npx gulp buildUWPPR
+      - name: Gulp Setup Project x86 (Windows)
+        run: npx gulp makeUWPProjectx86
+        working-directory: ./Package
+      - name: Gulp Build x86 Debug (Windows)
+        run: npx gulp buildUWPx86Debug
+        working-directory: ./Package
+      - name: Gulp NuGet Restore Playground
+        run: npx gulp nugetRestoreUWPPlayground
+        working-directory: ./Package
+      - name: Gulp Build x86 Debug Playground (Windows)
+        run: npx gulp buildUWPPlaygroundx86Debug
+        working-directory: ./Package
+
+  build-windows-x86-release:
```

Comment:
These all look like they are basically clones with configuration variations only, is that right? If so, can we leverage the GitHub workflow "matrix strategy" stuff? https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

Reply:
yes. i will look into this

Reply:
we now use a matrix for setup. Please let me know whether you are also wanting to reduce the number of gulp tasks/if you know of examples for providing arguments to gulp tasks